core/systems/input/input_watcher.gd

```diff
@@ -0,0 +1,61 @@
+@tool
+@icon("res://assets/editor-icons/game-icons--button-finger.svg")
+extends Node
+class_name InputWatcher
```

Comment:
This change also adds the `InputWatcher` node, which can be used in the future for responding to inputs.